service/policy/db/migrations/20250407000000_add_non_data_resources.sql

```diff
@@ -0,0 +1,51 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS non_data_resource_groups (
+  id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  name VARCHAR NOT NULL UNIQUE,
```

Comment:
Can you please add support in each of these tables for the typical policy object metadata key value labels here consistent across policy primitives?

Reply:
Done